### PR TITLE
Feature/orbit plugin

### PR DESCRIPTION
**Pull request by BrunoArrais**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,24 +36,24 @@ Seneca()
...
         }
       }
     }
   })
-  .use('tangocard-provider')
+  .use('orbit-provider')
 
-let board = await seneca.entity('provider/tangocard/board')
-  .load$('<tangocard-board-id>')
+let board = await seneca.entity('provider/orbit/board')
```
Comment (rjrodger) [inline README.md]: change this to an orbit specific example

File: src/orbit-provider.ts [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+    }
+  }
+
+
+  entityBuilder(this, {
+    provider: {
+      name: 'orbit'
+    },
+    entity: {
+      list_member: {
```
Comment (rjrodger) [inline src/orbit-provider.ts]: rename this to `member` and move the operations into the `list`,`save`,`load` operations - we need to follow the structure of REST operations

File: src/orbit-provider.ts [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+
+    if (!res.ok) {
+      throw this.fail('keymap')
+    }
+
+    this.shared.headers = {
+      Authorization: 'Bearer ' + res.keymap.key.value
+    }
+
+    setWorkspace(res.keymap.workspace.value)
```
Comment (rjrodger) [inline src/orbit-provider.ts]: refactor this to use `this.shared.workspace`

**Review by rjrodger — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (BrunoArrais) [reply README.md]: ok, swapped

File: src/orbit-provider.ts (hunk 2 above)
Comment (BrunoArrais) [reply src/orbit-provider.ts]: i believe now this correct

File: src/orbit-provider.ts (hunk 3 above)
Comment (BrunoArrais) [reply src/orbit-provider.ts]: ok, swapped

File: src/orbit-provider.ts [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+                return list
+              },
+          },
+          save: {
+            action: async function(this: any, entize: any, msg: any) {
+              let suffUrl =  this.shared.workspace + '/members/' + msg.q.idMember
+              let body = msg.q.body;
+              let res: any = await putJSON(makeUrl(suffUrl, msg.q), makeConfig(body))
+              
+              let list = entize(res)
```
Comment (rjrodger) [inline src/orbit-provider.ts:160]: returns the updated entity, not a list

File: src/orbit-provider.ts [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+              let body = msg.q.body;
+              let json: any = await postJSON(makeUrl(suffUrl, msg.q), makeConfig(body))
+              let data = json.data
+              let list = entize(data)
+              return list
+            },
+          }
+        }
+      },
+      list_member_by_organization: {
```
Comment (rjrodger) [inline src/orbit-provider.ts:189]: move into cmd.list.member as a special query

**Pull request merged**